packages/components/organisms/f-checkout/src/components/ErrorDialog.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+    <mega-modal
+        v-if="errorCode"
+        :is-open="isOpen"
```

Comment:
You have a `v-if` and an `is-open` int here and in checkout. Do they need to be in both places?